src/ZoneServer/Buffs/Handlers/Scout/Assassin/Behead_Debuff.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Network;
+using Melia.Zone.Skills;
+using Melia.Zone.Skills.Combat;
+using Yggdrasil.Logging;
+
+namespace Melia.Zone.Buffs.Handlers.Scout.Assassin
+{
+	/// <summary>
+	/// Buff handler for Behead: Bleeding, which deals damage
+	/// every 1.5 seconds
+	/// </summary>
+	[BuffHandler(BuffId.Behead_Debuff)]
+	public class Behead_Debuff : BuffHandler
+	{
+		public override void WhileActive(Buff buff)
+		{
+			if (!buff.Target.IsDead) { 
+				var skill = new Skill(buff.Caster, SkillId.Assassin_Behead_DOT, (int)buff.NumArg1);
```

Comment:
It seems like this skill's max level is 1, so there's presumably no point in setting its level.

Reply:
This is no longer necessary, though I disagree in general.  Even if the skill max level is 1 in the skillDb, levelling it up still works normally if the skill level is increased to be more than 1 (even things like factors display and work properly), so we should probably still handle skill levels correctly.

Reply:
While I fully agree with you that we should generally accommodate levels past the maximums, I don't believe this applies to `Assassin_Behead_DOT`, because it's a dummy skill without any additional effects. It doesn't do anything on its own and has no scaling factors whatsoever. As such, I would argue setting a level may cause confusion, making someone think there actually is more to it, when in reality there isn't and likely never will be.

Reply:
True, though we're not doing it anymore since we're no longer creating the skill object with the new overload.

Reply:
True, but it's something we can keep in mind going forward^^